Classes/OldModel/Event.php

```diff
@@ -4606,20 +4606,18 @@ public function createImageForSingleView($maxImageWidth, $maxImageHeight)
                 'height' => $maxImageHeight,
             ],
         ];
-        $imageWithTag = $this->cObj->cObjGetSingle('IMAGE', $imageConfiguration);
+        $imageTag = $this->cObj->cObjGetSingle('IMAGE', $imageConfiguration);
 
         $imageWidth = [];
-        preg_match('/width="([^"]*)"/', $imageWithTag, $imageWidth);
+        preg_match('/width="([^"]*)"/', $imageTag, $imageWidth);
         $imageHeight = [];
-        preg_match('/height="([^"]*)"/', $imageWithTag, $imageHeight);
+        preg_match('/height="([^"]*)"/', $imageTag, $imageHeight);
         $imageUrl = [];
-        preg_match('/src="([^"]*)"/', $imageWithTag, $imageUrl);
+        preg_match('/src="([^"]*)"/', $imageTag, $imageUrl);
```

Comment:
We don't need all this `preg_match` anymore - `$imageTag` already will have the finished tag. (Those `preg_match` were used to extract the data from the image tag, which we don't need anymore.)